modules/python/src2/gen2.py

```diff
@@ -300,6 +302,8 @@ def __init__(self, name, decl=None, codegen=None):
                     self.ismap = True
                 elif m == "/Simple":
                     self.issimple = True
+                elif m == "/Params":
+                    self.is_parameters = True
```

Comment:
Looks like this change is missing:
https://github.com/opencv/opencv/blame/23dec329b47cf19ce2bbbb623c1467929ac021e3/modules/python/src2/gen2.py#L307